source/MRVoxels/MRDicom.cpp

```diff
@@ -654,9 +657,6 @@ Expected<DicomVolumeT<T>> loadSingleDicomFolder( std::vector<std::filesystem::pa
     presentSlices.resize( data.dims.z );
     presentSlices.flip();
 
-    if constexpr ( std::convertible_to<T, VdbVolume> )
-        data.data->denseFill( toVdbBox( data.dims ), 0 );
```

Comment:
Dont we lose performance by manualy densifing vdb grid?

Reply:
Otherwise it is unsafe to use accessors from different threads, so we don't have a choice.